ui/job-view/details/BugFiler.jsx

```diff
@@ -264,25 +264,27 @@ export class BugFilerClass extends React.Component {
       if (lowerJobGroupName.includes('web platform')) {
         failurePath = failurePath.startsWith('mozilla/tests')
           ? `testing/web-platform/${failurePath}`
-          : `testing/web-platform/tests/${failurePath}`;
+          : `testing/web-platform/tests${failurePath}`;
       }
 
       // Search mercurial's moz.build metadata to find products/components
       fetch(
         `${hgBaseUrl}mozilla-central/json-mozbuildinfo?p=${failurePath}`,
       ).then(resp =>
         resp.json().then(firstRequest => {
+          if (firstRequest.error) {
```

Comment:
moz.build seems to be returning an error these days instead of finding product suggestions. Just notify in this case so the user knows to manually search for a product.